### DISCUSSION TITLE
Airflow connections --conn-extra parameter missing_host_key_policy not taking in effect - v2.5.1

### DISCUSSION THREAD

**Discussion by jayramr** [apache/airflow#30436] [General]
Hello, 

I'm using Airflow v2.5.1 based on pip installation. I defined the connection parameters as below 

```
airflow connections add  test_ssh   --conn-type ssh \
    --conn-host test.com \
    --conn-login test  \
    --conn-port 4532 \
    --conn-extra '{"key_file": "/Users/jr5241/.ssh/id_rsa", "missing_host_key_policy": "AutoAddPolicy"}'
```

Then i test this connection using Airflow connections tab page and checked the logs and I can see

```
[2023-04-03 11:47:32,280] {base.py:73} INFO - Using connection ID 'o3m08wGg' for task execution.
[2023-04-03 11:47:32,283] {ssh.py:299} WARNING - No Host Key Verification. This won't protect against Man-In-The-Middle attacks
[2023-04-03 11:47:32,360] {transport.py:1871} INFO - Connected (version 2.0, client OpenSSH_8.9p1)
```

Here the connection is established successfully, but I'm bit worried about this host key verfication warning. 

Also I tried to add the ssh connection programatically from the plugin as follows.

```
import paramiko
ssh = paramiko.SSHClient()
ssh.set_missing_host_key_policy(paramiko.AutoAddPolicy())
ssh.connect(hostname='test.com', username='test')
```
But there is no warning for the above . Kindly advise.

Thanks
Jay

**Comment by potiuk** [comment]
Look there: https://airflow.apache.org/docs/apache-airflow-providers-ssh/stable/connections/ssh.html
